scipy/stats/_continuous_distns.py

```diff
@@ -10286,6 +10286,140 @@ def _single_cdf(q, k, df):
                                           b=np.inf)
 
 
+class relativistic_bw_gen(rv_continuous):
+    r"""A relativistic Breit-Wigner random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    cauchy: Cauchy distribution, also known as the Breit-Wigner distribution.
+
+    Notes
+    -----
+
+    The probability density function for `relativistic_bw` is
+
+    .. math::
+
+        f(x, \rho) = \frac{k}{(x^2 - \rho^2)^2 + \rho^2}
+
+    where
+
+    .. math::
+        k = \frac{2\sqrt{2}\rho^2\sqrt{\rho^2 + 1}}
+            {\pi\sqrt{\rho^2 + \rho\sqrt{\rho^2 + 1}}}
+
+    The relativistic Breit-Wigner distribution is used in high energy
+    physics to model resonances. It gives the distribution for the observed
+    center-of-mass energy produced by a resonance given its mass :math:`M` and
+    decay-width :math:`\Gamma` [1]_, where :math:`M` and :math:`\Gamma` are
+    expressed in natural units. In SciPy's parametrization, the shape parameter
+    :math:`\rho` is equal to :math:`M/\Gamma`.
+
+    %(after_notes)s
+
+    :math:`\rho = M/\Gamma` and :math:`\Gamma` is the scale parameter. If one
```

Comment:
"For example, if one..." maybe?

Reply:
I’d dithered on this one. Let’s go with your suggestion.